_includes/sections/hosting-provider.html

```diff
@@ -1,5 +1,9 @@
 <h1 id="host" class="anchor"><a href="#host"><i class="fas fa-link anchor-icon"></i></a> Secure Hosting Provider</h1>
 
+<div class="alert alert-warning" role="alert">
+  <strong>Note: We cannot know what the hosting providers are doing and have to take their word on it.</strong>
```

Comment:
What about adding an additional suggestion to verify their privacy policies?
```suggestion
  <strong>Note: We cannot know what the hosting providers are doing and have to take their word on it. Please verify their privacy policies.</strong>
```

Reply:
I committed this, but would "read" make more sense than "verify" ?

Reply:
I meant verifying their claims on their main home pages, but yeah “read” probably is more clear.

Reply:
Changed to read, please rereview